versions.yml

```diff
@@ -2,6 +2,16 @@
 logstash: 7.0.0-alpha1
 logstash-core: 7.0.0-alpha1
 logstash-core-plugin-api: 2.1.16
+
+# jruby must reference a *released* version of jruby which can be downloaded from the official download url
+# *and* for which jars artifacts are published for compile-time
 jruby:
   version: 9.1.12.0
   sha1: 9b6c15d42eb11db7215d94dd9842ee5184a41fea
+
+# jruby-runtime, if specified, will override the jruby version installed in vendor/jruby for logstash runtime only,
+# not for the compile-time jars
+jruby-runtime:
+  url: http://ci.jruby.org/snapshots/previous/jruby-bin-9.1.13.0-SNAPSHOT.tar.gz
+  version: 9.1.13.0-SNAPSHOT
+  sha1: 47560ccae3e30f8193cb5b40b8deef4a39fa094e
```

Comment:
Won't this break any time they publish a new snapshot with the same filename? (The checksum will be different)

Reply:
good point.